config/environments/test.rb

```diff
@@ -62,11 +62,14 @@
   # Feature Toggle for UMM-S
   config.umm_s_enabled = true
 
+  # Feature Toggle for UMM-Var Generation
+  config.uvg_enabled = true
```

Comment:
this is what bamboo uses I am assuming, we on the product side will need to keep this in mind.

Reply:
Yes, this is what will be used for our tests locally and in Bamboo